src/fiat/funds-transfers.md

```diff
@@ -125,3 +126,113 @@ curl -X GET "https://service.centrapay.com/api/topups" \
   }
 ]
 ```
+
+## Creating a Withdrawl **EXPERIMENTAL**
+
+{% endpoint POST https://service.centrapay.com/api/withdrawls %}
+
+```sh
+curl -X POST "https://service.centrapay.com/api/withdrawls" \
+  -H "x-api-key: 1234" \
+  -H "content-type: application/json" \
+  -d '{
+    "amount": "10000",
+    "walletId": "Te2uDM7xhDLWGVJU3nzwnh",
+    "bankAuthorityId": "FRhAzzWpTKb5U7pZygQjjY"
+  }'
+```
+
+**Required Fields**
+
+|      Field      |  Type  |               Description                |
+| :-------------- | :----- | :--------------------------------------- |
+| amount          | String | Total amount of the transaction in cents |
+| walletId        | String | The id of the wallet                     |
+| bankAuthorityId | String | The id of the bank authority             |
+
+**Example response payload**
+
+```json
+{
+  "id": "hg2RfYTQ635tPBZEPJdCre",
+  "walletId": "Te2uDM7xhDLWGVJU3nzwnh",
+  "bankAuthorityId": "FRhAzzWpTKb5U7pZygQjjY",
+  "accountId": "aBc932S9182qwCDqwer",
+  "type": "withdrawl",
+  "amount": "10000",
+  "status": "created",
+  "createdAt": "2020-05-01T12:30:00.000Z",
+  "updatedAt": "2020-05-01T12:30:00.000Z"
+}
+```
+
+**Error Responses**
+
+| Status |                     Code                      |                                         Description                                         |
+| :----- | :-------------------------------------------- | :------------------------------------------------------------------------------------------ |
+| 403    | {% break _ BANK_AUTHORITY_WALLET_MISMATCH %}  | The wallet and the bank authority for the top up request do not belong to the same account. |
+| 403    | {% break _ SPENDING_MONTHLY_LIMIT_EXCEEDED %} | The user has gone over their quota for maximum withdawls this month.                        |
+| 403    | {% break _ SPENDING_YEARLY_LIMIT_EXCEEDED %}  | The user has gone over their quota for maximum withdawls this year.                         |
+
+## Get a withdrawl by id **EXPERIMENTAL**
+
+{% endpoint GET https://service.centrapay.com/api/withdrawals/${id} %}
+
+```sh
+curl -X GET "https://service.centrapay.com/api/withdrawals/WRhAxxWpTKb5U7pXyxQjjY" \
+  -H "x-api-key: 1234"
+```
+
+**Example response payload**
+
+```json
+{
+  "id": "hg2RfYTQ635tPBZEPJdCre",
+  "walletId": "Te2uDM7xhDLWGVJU3nzwnh",
+  "bankAuthorityId": "FRhAzzWpTKb5U7pZygQjjY",
+  "accountId": "aBc932S9182qwCDqwer",
+  "type": "withdrawl",
+  "amount": "10000",
+  "status": "created",
+  "createdAt": "2020-05-01T12:30:00.000Z",
+  "updatedAt": "2020-05-01T12:30:00.000Z"
+}
+```
+
+## List withdrawls for authorized accounts **EXPERIMENTAL**
+
+{% endpoint GET https://service.centrapay.com/withdrawls %}
```

Comment:
/api/withdrawals

Reply:
I don't want us to make new endpoints that query for all authorized accounts. Can we put the account id as a path param?

Reply:
This API should be forwards compatible with pagination. Can you please use paginated response schema?

{ items: [] }

Reply:
Whoops, updated!

Reply:
> Whoops, updated!

^^ this was in the context of `/api/withdrawls`.

I'll move path into the URL. Should we create endpoints to support topup flows in the same way?

I'll go add  a paginated response.

Reply:
Hey @ncjones , I've gone and updated the response and endpoints to be more account specific. I've also gone and added an EXPERIMENTAL version for topups so we have symmetry.